reconcile/utils/ocm/status_board.py

```diff
@@ -117,6 +118,18 @@ def create_service(ocm_api: OCMBaseClient, spec: ServiceOCMSpec) -> str:
     return resp["id"]
 
 
+def update_application(
+    ocm_api: OCMBaseClient,
+    application_id: str,
+    spec: ApplicationOCMSpec,
+) -> None:
+    data = spec | {
+        "metadata": spec.get("metadata", {})
+        | {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE}
+    }
```

Comment:
now `METADATA_MANAGED_BY_KEY` is managed by caller, do we still need to overwrite here?